packages/core/content-manager/admin/src/pages/ListView/ListViewPage.tsx

```diff
@@ -330,6 +329,30 @@ const ActionsCell = styled(Table.Cell)`
   justify-content: flex-end;
 `;
 
+/* -------------------------------------------------------------------------------------------------
+ * TableActionsBar
+ * -----------------------------------------------------------------------------------------------*/
+
+const TableActionsBar = () => {
+  const selectRow = useTable('TableActionsBar', (state) => state.selectRow);
+  const [{ query }] = useQueryParams<{ plugins: { i18n: { locale: string } } }>();
+  const locale = query?.plugins?.i18n?.locale;
+  const prevLocale = usePrev(locale);
+
+  // TODO: find a better way to reset the selected rows when the locale changes across all the app
```

Comment:
Have you looked into handling the event when the user clicks on the locale dropdown?

Reply:
Yes its in i18n plugin, so making changes from this plugin inside Table is not doable.
I thought of making this change inside `Table.tsx` but I am not sure if it will pollute the Table functionality.

Reply:
I see, in that case in think your solution is fine 👍